Sprint-2/debug/1.js

```diff
@@ -1,8 +1,7 @@
 // Predict and explain first...
```

Comment:
Predict what the program will do and write down that prediction. 

Then run it. Then explain what happened. 

You must focus on understanding and explaining.

Reply:
Dear @SallyMcGrath ,
form now on I'll never forget to: Predict,Run,Explain 👍